hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -184,15 +186,19 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
           // Differentiate key without trailing slash
           return (subpath.isKey || subpathName === OTHER_PATH_NAME) ? subpathName : subpathName + '/';
         });
-
+         
         values = subpaths.map(subpath => {
           return subpath.size / dataSize;
         });
-
+        // Logic for Values Normalization on Pie chart adding 0.05 so minimum block
+        // will be displayed on Pie chart using below logic
+        // Percentage and Size string Logic will be same 
```

Comment:
Change the comment to the following :- 
```suggestion
    // Normalize values for the pie chart to ensure better visual representation.
    // Adding a small constant (MIN_BLOCK_SIZE) to each normalized value ensures that even 
    // the smallest entities are visible on the pie chart.
    // Note: The percentage and size string calculations remain unchanged.
```